dockerfiles/centos/is/Dockerfile

```diff
@@ -17,9 +17,9 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK CentOS Docker image
-FROM adoptopenjdk/openjdk11:x86_64-centos-jdk-11.0.8_10
```

Comment:
In the case of this architecture addition x86_64, shall we maintain the same pattern used for the other simultaneous releases?

Reply:
Yes +1. Let's avoid using the architecture. IIUC, this allows the image to be built according to the platform architecture in the given context.